query/src/sql/sql_parser.rs

```diff
@@ -346,6 +346,8 @@ impl<'a> DfParser<'a> {
             Ok(DfStatement::ShowSettings(DfShowSettings))
         } else if self.consume_token("CREATE") {
             self.parse_show_create()
+        } else if self.consume_token("FIELDS") {
```

Comment:
> } else if self.consume_token("FIELDS") {

} else if self.consume_token("FIELDS") || self.consume_token("COLUMNS") {

maybe we can also support `columns`?
```SQL
SHOW [EXTENDED] [FULL] {COLUMNS | FIELDS}
    {FROM | IN} tbl_name
    [{FROM | IN} db_name]
    [LIKE 'pattern' | WHERE expr]
```

Reply:
Good idea, will try it later in this PR

Reply:
Oh, I have a check, this syntax is not support LIKE and only for mysqldump not for user in databend now.